src/main/scala/temple/generate/language/service/go/GoServiceGenerator.scala

```diff
@@ -0,0 +1,56 @@
+package temple.generate.language.service.go
+
+import temple.generate.language.service.ServiceGenerator
+import temple.generate.language.service.adt._
+import temple.utils.FileUtils.{File, FileContent}
+import temple.utils.StringUtils.tabIndent
+
+object GoServiceGenerator extends ServiceGenerator {
+
+  private def generatePackage(packageName: String): String = s"package $packageName\n\n"
+
+  private def generateImports(serviceName: String, module: String, comms: Boolean): String = {
+    val sb = new StringBuilder
+
+    sb.append("""import (
```

Comment:
Is there any way we can avoid the parens being separated? Could these strings be wrapped in quotes, separated by new lines, and indented, then we call codeParens on them?